### DISCUSSION TITLE
customizing container max-width variants

### DISCUSSION THREAD

**Discussion by syffs** [unocss/unocss#1290] [Q&A]
is it possible to update container class `@media` variants, to customize `max-width` ? If so, how ?

AFAIK you can set the `@media` queries conditions with [tailwind through its config file](https://tailwindcss.com/docs/screens#max-width-breakpoints), not sure how you can set up the underlying css rules.

**Comment by zyyv** [comment]
You just need custom your breakpoints in theme option. Then use `lt` variant can be achieved. 
[Extend-theme](https://github.com/unocss/unocss#extend-theme)

[playground](https://uno.antfu.me/play/?html=DwEwlgbgBAtiBcAXApgD0QWgE7JAPgFcA7AewGMBnC4AenAjwChRIoBDTOJNTHfY8lVr0mLaABtOCFOmy5CpStTqQ8QA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwod6gBccADawRrCbFQwagC0fraoGmRIorbQQRqUyBqKALq5JN5wVAAW0DARMFRBXj4%2B4ZHRIDGl5ZUaGTCmMDG2RHBdJj3QVlhEqHBlAG6oUAGDPV1WtklFCoU%2BMCWoIKg1RT4slFYA1pDYVft1dSbVmgDMAEwADGAmq9c%2BVCAZAGwALK93htPiBkBkAOy-AAcbw%2Bnz6GQAjM9Hv84SDriZbMjHtCgfDrhpHtiOpokQBWe6-DEHG7Y5FUmnAunrNYg-hSO7BOmc2iyCDuACUmPIkloDCYrA4XG4wtFfJgIjEnjpX2Eyz2cCRADpHqKfMJkFgMiVGGBqgB6S3UEA60qWwmKEVFAoGBRCgxAA&options=N4XyA)

**Reply by syffs** [reply]
Thanks for the quick reply !

The thing is I am ok with breakpoints current values, I don't want to change them for the whole app: I only want `container` class to have different/lower `max-width` rules in bigger breakpoints
